calcom/cal.com
# fix: [CAL-3940] randomize Tip Sidebar

## What does this PR do?
Fixes #15458 

Used lodash shuffle to randomize "Tips Sidebar"


Loom Video: https://www.loom.com/share/6f839cc1435e4b5d8afb535b2ce34ad5?sid=26474013-c39e-4722-8511-9a20dc7e4148

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?


- Are there environment variables that should be set?
NO
- What are the minimal test data to have?
NA
- What is expected (happy path) to have (input and output)?
To see randomized Tips sidebar on each reload
- Any other important info that could help to test that PR
- No

## Checklist
